parallel_execute_scripts/HowTo.md

```diff
@@ -0,0 +1,44 @@
+## Step 1. Create the Loader and execute scripts. 
```

Comment:
Should we name this Readme.md -- I think that is more standard and it works better with Github.

Reply:
Done.